rt_sensorlevelanalysis.m

```diff
@@ -33,16 +33,23 @@
   end
 end
 
-if isempty(cfg)
+if isempty(cfg) && ~strcmp(type, 'tfr')
   cfg.lpfilter   = 'yes';
   cfg.lpfreq     = 35;
   cfg.lpfilttype = 'firws';
+elseif isempty(cfg) && strcmp(type, 'tfr')
+  cfg.lpfilter = 'no';
+  latency      = [-1 2-1./300];
+end
+
+if ~exist('latency', 'var')
+  latency = [];
 end
 
 root_dir = '/project/3012026.13/processed';
-data     = rt_mytimelockv3(root_dir, subj, 'cfg_preproc', cfg);
+data     = rt_mytimelockv3(root_dir, subj, 'cfg_preproc', cfg, 'latency', latency);
 data     = ft_appenddata([], data{:});
-data     = removefields(data, {'elec'});
+data     = removefields(data, {'elec' 'cfg'});
```

Comment:
In wonder why this field needs to be removed.

Reply:
currently, the cfg takes up a lot of space, due to the toggling back and forth in my_timelockv3 (essentially it gets about 72 copies of itself in cfg.previous.previous somewhere.

the elec is not used, and slows down the downstream computations.

Reply:
Got it, thanks!